app/helpers/hyku_addons/api_works_helper.rb

```diff
@@ -8,16 +8,13 @@ def creator_from_json(creator_json)
 
       return if creator_hash.blank?
 
-      key = "creator_institutional_email"
```

Comment:
You've remove the variable and are now repeating your variable string multiple times.

Reply:
And Rubocop is no longer shouting at me about complexity. You rather I just ignore the Cop?

Reply:
Rubocop isn't God :) 

But you shouldn't make code worse just to appease it. Rubocop uses the number of `.`'s (and other stuff like if statements) to determine the potential complexity. So by moving the `user` query out to a helper method you can also save complexity there.